docs/docs/mail.md

```diff
@@ -1,345 +1,186 @@
-   adjusted in complex ways, thus invalidating most of the benefits.
-
-We'll now explore these issues in more depth.
-
-### TCB size
-
-The code that handles encrypted messaging is directly exposed to attackers. It is important to minimise
-the amount of code exposed in this way. For more discussion of how enclaves are designed to minimise code size, 
-please see ["Small is beautiful"](enclaves.md#small-is-beautiful).  
-
-### TLS complexity
-
-Mail is based on the [Noise protocol framework](https://noiseprotocol.org/). Noise is designed by professional
-cryptographers and is used in WhatsApp, WireGuard and other modern network protocols. Noise was created because TLS
-is weighed down with complicated historical baggage and web-specific features. A Noise protocol delivers most of
-the same functionality but in a more modular, cleaner and simpler way. Therefore, outside of the web the primary
-reason to use TLS is compatibility, not technical excellence. Now there is Noise there isn't much reason to use TLS
-anymore when designing a new protocol from scratch, unless you really need its features or support.
-
-TLS 1.3 recognises this problem and simplifies the protocol, but it's still far more complex than a Noise protocol, 
-and because the purpose of using TLS is compatibility and TLS 1.3 is very new, realistically you will support TLS 1.2 
-as well. So now you have even more complexity in your enclave because you need both 1.2 and 1.3 support 
-simultaneously.
-
-A big source of complexity in TLS is X.509 certificates. Certificates do not make sense for an enclave but TLS 
-absolutely requires them, so some systems try to jam fake pseudo-certificates into TLS, which aren't real but
-contain the remote attestation in an extension field. Fake certificates can create problems because a lot of 
-organisations have decades of procedures and best practices around all aspects of certificates and X.509, 
-so fake pseudo-certificates can end up harder to deploy and work with than a purely technical analysis would suggest.
-Noise is more modular - you can provide any arbitrary byte array as part of the handshake, which can thus 
-include a certificate (of any format) if you want that, but it isn't necessary.
-
-In Conclave the "certificate" is the remote attestation data represented as an [`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object. 
-It will usually make sense to expose this to the client via some app-specific mechanism, for example, returning it
-in some other API, publishing it on a web server, a network drive, a message queue, even putting it into a 
-distributed hash table. Noise and by extension Conclave doesn't care how you get this data, only that you have it.
+The Conclave runtime verifies that the sequence numbers under a topic always increment by one before passing a Mail 
+item to the enclave. This feature ensures that the host can't reorder messages or drop Mail items.
```

Comment:
the host can't reorder or drop Mail messages.

Reply:
Corrected in [another PR](https://github.com/R3Conclave/conclave-core-sdk/pull/58).